manifests/init.pp

```diff
@@ -835,6 +924,8 @@
     create_resources('pam::limits::fragment',$limits_fragments)
   }
 
+  validate_absolute_path($password_auth_ac_file)
```

Comment:
also need to validate `$password_auth_file` and add the corresponding spec test for failure